.travis.yml

```diff
@@ -152,24 +151,27 @@ base_acceptance: &acceptance_default
 matrix:
   fast_finish: true
   include:
-    # Lint python and javascript together
     - python: 2.7
-      name: 'Linter'
-      env: TEST_SUITE=lint
+      name: 'Linter (Javascript)'
```

Comment:
\o/ do we need to define python version in line above?

Reply:
Not anymore, good catch! I usually don't miss these kinds of things.